app/controllers/likes_controller.rb

```diff
@@ -43,6 +44,7 @@ def update
       else
         format.html { render :edit, status: :unprocessable_entity }
         format.json { render json: @like.errors, status: :unprocessable_entity }
+        format.js
```

Comment:
I don't think you need format.js in your update method for your likes. You can either like or unlike so there is no update more of a create or destroy.

Reply:
I just fixed it. Thx!